lib/modules/datasource/azure-pipelines-tasks/index.ts

```diff
@@ -22,13 +25,70 @@ export class AzurePipelinesTasksDatasource extends Datasource {
   async getReleases({
     packageName,
   }: GetReleasesConfig): Promise<ReleaseResult | null> {
-    const versions =
-      (await this.getTasks(BUILT_IN_TASKS_URL))[packageName.toLowerCase()] ??
-      (await this.getTasks(MARKETPLACE_TASKS_URL))[packageName.toLowerCase()];
+    const platform = GlobalConfig.get('platform');
+    const endpoint = GlobalConfig.get('endpoint');
 
-    if (versions) {
-      const releases = versions.map((version) => ({ version }));
-      return { releases };
+    if (platform === 'azure' && endpoint) {
+      const auth = Buffer.from(
+        `renovate:${process.env.RENOVATE_TOKEN}`,
```

Comment:
It's not certain that RENOVATE_TOKEN is defined. Instead, use `hostRules` to get the token

Reply:
I have used the GlobalConfig and hostRules based on other data sources but I don't know if I have done it right.

I don't see any API documentation, I've only seen the pages you recommend here: https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#code

Reply:
@rarkins can you answer please?